modules/update-upgrading-cli.adoc

```diff
@@ -48,10 +34,10 @@ $ oc adm upgrade
 .Example output
 [source,terminal]
 ----
-Cluster version is 4.9.23
+Cluster version is 4.{product-version}.23
 
 Upstream is unset, so the cluster will use an appropriate default.
-Channel: stable-4.10 (available channels: candidate-4.10, candidate-4.9, fast-4.10, fast-4.9, stable-4.10, stable-4.9)
+Channel: stable-4.9.23 (available channels: candidate-4.9.23, fast-4.9.23, stable-4.9.23, eus-*)
```

Comment:
Channels are just 4.y, like `stable-4.9`, not `stable-4.9.23`, and similar for the other channels.  We'll also need to use a 4.10 channel  to get the 4.10.13, etc. examples in the following `Recommended updates` section, so we might want to stick with `stable-4.10` as the selected channel  here.  Or we can move it to `stable-4.9` and then drop all the 4.10.z targets below.

Reply:
@wking would this mean adjusting stable-4.9 to stable-4.10 and removing the 4.9.x examples from the example output?